modules/shredder/src/main/scala/com.snowplowanalytics.snowplow.storage/spark/ShredJobConfig.scala

```diff
@@ -46,7 +46,8 @@ case class ShredJobConfig(inFolder: String,
                           duplicateStorageConfig: Option[Json],
                           dynamodbManifestTable: Option[String],
                           itemId: Option[String],
-                          storage: Option[StorageTarget]) {
+                          storage: Option[StorageTarget],
+                          sqsQueue: String) {
```

Comment:
SQS queue must be passed with `StorageTarget` (unless you want to implement this logic in EmrEtlRunner), i.e. if there's no SQS queue in the config (or even config itself), both Shredder and Loader maitain legacy behavior, otherwise they both communicate through this channel.